src/Symfony/Component/Finder/Finder.php

```diff
@@ -153,6 +155,30 @@ public function date($dates)
         return $this;
     }
 
+    /**
+     * Adds rules that files must match like $finder->name() but with case-insensitivity set.
+     *
+     * You can use patterns (delimited with / sign), globs or simple strings.
+     *
+     *     $finder->iname('*.php') // will match both *.php and *.PHP
+     *     $finder->iname('/\.php$/i') // same as above
```

Comment:
Wait, this means we can already achieve case insensitivity right? which means we don't need `iname()`!

Reply:
Yes, that's right. If you use a regex, you can achieve case insensitivity with `$finder->name(...)`
This possible solution was described in the description of #28503.

Reply:
I would prefer to not add `iname()` as it makes things more complex for something we can already do with the current `name()` method.